Service/NotificationService.php

```diff
@@ -154,15 +157,9 @@ public function setStatus(Notification $notification, $status, $level = 'notice'
     
     /**
      * Call addNotification when click on a request
-     *
-     * @param unknown $template
-     * @param unknown $revision
      */
-    public function addNotification($templateId, $revision, $environment)
+    public function addNotification(string $templateId, Revision $revision, Environment $environment)
```

Comment:
code calling this function is compatible with strong typing (string, Revision & Environment)

Reply:
I think that a template id id an integer, not a string. Or it's a string but converted to a integer somewhere

Reply:
@theus77 
Looks like there is a problem on line 170:
$template = $repository->findOneById($templateId);

The function findOneById does not exist in the repositories.

Reply:
Nvm i found the magic in __call of EntityRepository (doctrine)